src/GoTrueClient.ts

```diff
@@ -1720,9 +1762,16 @@ export default class GoTrueClient {
           if (expiresInTicks <= AUTO_REFRESH_TICK_THRESHOLD) {
             await this._callRefreshToken(session.refresh_token)
           }
-        })
+        }, 0 /* try-lock */)
       } catch (e: any) {
-        console.error('Auto refresh tick failed with error. This is likely a transient error.', e)
+        if (e.isAcquireTimeout) {
+          this._debug(
+            '#_autoRefreshTokenTick()',
+            'lock is already acquired, skipping for next tick'
+          )
+        } else {
+          console.error('Auto refresh tick failed with error. This is likely a transient error.', e)
```

Comment:
What would be the best way to advise a customer to debug this if they come back to us with an error?

Reply:
It's transient, but logging so it's not silent.